<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QDialog" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox
{
border: 1px solid;
border-radius: 12px; 
border-color: rgba(33,35,50,255);
color:#0000ad;
	background-color: rgb(255, 255, 255);
}
*{
	background-color: rgba(33,35,50,255);
font-family:Ubuntu Medium;
color:#fff;
}
QLineEdit
{
height:40px;
border: 1px solid;
border-color: rgba(33,35,50,255);
border-radius:10px;
color:rgba(33,35,50,255);
background-color: rgb(255, 255, 255);
}
QPushButton
{
background-color: #a7abaa;
color: rgba(0,0,173,255);
border-radius: 12px; 
border: 1px solid;
border-color: rgba(33,35,50,255);
height:40px;
width:100px;
}
QGroupBox
{
	width:500px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border: 1px solid;
border-radius: 12px; 
border-color: rgba(33,35,50,255);
color:#0000ad;
	background-color: rgb(255, 255, 255);
}
*{
	background-color: rgba(33,35,50,255);
font-family:Ubuntu;
color:#fff;
}
QLineEdit
{
height:40px;
border: 1px solid;
border-color: rgba(33,35,50,255);
border-radius:10px;
color:rgba(33,35,50,255);
background-color: rgb(255, 255, 255);
}
QPushButton
{
background-color: #a7abaa;
color: rgba(0,0,173,255);
border-radius: 12px; 
border: 1px solid;
border-color: rgba(33,35,50,255);
height:40px;
width:100px;
}
QGroupBox
{
	width:500px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>55</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;font color = #fff&gt;Change Password
</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="Login">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#fff;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="id">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>&lt;font color = #fff&gt;ID</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="idLineEdit">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: 1px solid;
border-color: rgb(255, 111, 42);
border-radius:5px;
height:45px;
background-color: rgb(255, 255, 255);
color:#0000ad;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="password">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>&lt;font color = #fff&gt;Old Password
</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Preferred</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="currentPasswordLineEdit">
                 <property name="font">
                  <font>
                   <family>Ubuntu</family>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid;
border-color: rgb(255, 111, 42);
border-radius:5px;
height:45px;
color:#0000ad;</string>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Normal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>New Password</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="newPasswordLineEdit">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid;
border-color: rgb(255, 111, 42);
border-radius:5px;
height:45px;
color:#0000ad;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Confirm Password</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="confirmPassword">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid;
border-color: rgb(255, 111, 42);
border-radius:5px;
height:45px;
color:#0000ad;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="changePasswordButton">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #a7abaa;
color: rgba(0,0,173,255);
border-radius: 12px; 
border: 1px solid;
border-color: #0000ad;
height:50px;
margin-left:80px;
margin-right:80px;</string>
             </property>
             <property name="text">
              <string>Change Password</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #a7abaa;
color:rgba(0,0,173,255);
border-radius: 12px; 
border: 1px solid;
border-color: #0000ad;
height:50px;
margin-left:80px;
margin-right:80px;</string>
             </property>
             <property name="text">
              <string>Go back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
